docs/basic_configuration.rst

```diff
@@ -0,0 +1,66 @@
+There are much more options, see [[configuration]].
+
+Preparations
```

Comment:
You probably don't need a `s` at the end of this.

Reply:
Indeed. This was copied from the wiki.